tests/test_user_views.py

```diff
@@ -57,38 +50,43 @@ def test_user_signup(self):
         user.id = uid
 
         db.session.commit()
+        with self.client as c:
+            resp = c.post('/login', data={'username': 'test1234',
+                                          'password': 'password'},follow_redirects=True)
+            html = resp.get_data(as_text=True)
+            self.assertEqual(resp.status_code, 200)
+            self.assertIn("dummytest2@test.com", html)
+
+    def test_invalid_login(self):
+        """Does authentication fail with invalid credentials are provided?"""
+        with self.client as c:
+            resp = c.post('/login', data={'username': self.testuser.username,
+                                          'password': self.testuser.password},follow_redirects=True)
 
-        with self.client as client:
-            resp = client.post('/register', data={'firstname': 'tester',
-                                                  'lastname': 'person',
-                                                  'username': 'test12345',
-                                                  'email': 'dummytest22@test.com',
-                                                  'password': 'password',
-                                                  'image': None,
-                                                  'state': 'Texas',
-                                                  'vax_date': None,
-                                                  'covid_status': None},
-                                                   follow_redirects=True)
             html = resp.get_data(as_text=True)
             self.assertEqual(resp.status_code, 200)
-            self.assertIn("dummytest22@test.com", html)
-
-    def test_invalid_user_signup(self):
-        with self.client as client:
-            resp = client.post('/register', data={'firstname': 'tester',
-                                                  'lastname': 'person',
-                                                  'username': 'test123',
-                                                  'email': 'test@test.com',
-                                                  'password': 'password',
-                                                  'image': None,
-                                                  'state': 'Texas',
-                                                  'vax_date': None,
-                                                  'covid_status': None},
-                                                   follow_redirects=True)
```

Comment:
@azaria-dedmon why are we removing this test ?

Reply:
This is not related to login/logout testing.

Reply:
@azaria-dedmon Note that a PR is essentially an edit of what already exists in the master branch. When we add or remove something in a PR and it is merged, those changes will take effect in the master branch. And since this PR is only meant to add login/logout functionality, we should not be removing the registration functionality.